core/session_test.go

```diff
@@ -104,10 +104,8 @@ func TestSessionActivate(t *testing.T) {
 	session := NewSession()
 	user := &User{Username: DefaultUserName}
 
-	err := session.Activate(user)
-	if err != nil {
-		t.Errorf("Session.Activate() returned an error: %v", err)
-	}
+	session.Activate(user)
+
```

Comment:
Why is this change needed?

Reply:
<img width="565" alt="image" src="https://github.com/user-attachments/assets/70fd78e0-6990-4f09-955e-ba2349d96a8a">

My editor was complaining about this issue and so I made this change. I understand it is not related to the PR, would like me to remove it?